### PR TITLE
Enable automation test for featurestore samples

### PR DESCRIPTION
**Pull request by fredms**
# Description
Enable automation test for featurestore samples

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
- [x] I have coordinated with the docs team (mldocs@microsoft.com) if this PR deletes files or changes any file names or file extensions.
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by qjxu — APPROVED** [review]

File: sdk/python/featurestore_sample/setup-resources.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
+pip install --upgrade jupytext
+
+# <create_variables>
+SUBSCRIPTION_ID=$(az account show --query id -o tsv)
+LOCATION=$(az ml workspace show --query location -o tsv)
+RESOURCE_GROUP=$(az group show --query name -o tsv)
+AML_WORKSPACE_NAME=$(az configure -l --query "[?name=='workspace'].value" -o tsv)
```
Comment (bastrik) [inline sdk/python/featurestore_sample/setup-resources.sh:7]: Will workspace be the same for each run?

Comment (fredms) [reply sdk/python/featurestore_sample/setup-resources.sh:7]: yes, will be the same for a month..

Comment (bastrik) [reply sdk/python/featurestore_sample/setup-resources.sh:7]: I see, feature store today does not allow asset re-registration over the same version. So we would need to find a way to change the asset version each time the notebook is run

**Review by bastrik — APPROVED** [review]

File: sdk/python/featurestore_sample/setup-resources.sh (hunk 1 above)
Comment (bastrik) [reply sdk/python/featurestore_sample/setup-resources.sh:7]: Discussed offline, will use dynamic versioning when we enable rest of the notebooks

File: sdk/python/featurestore_sample/notebooks/sdk_only/2. Enable materialization and backfill feature data.ipynb [hunk 2]
```diff
@@ -159,7 +159,7 @@
     "\n",
     "# please update the dir to ./Users/{your-alias} (or any custom directory you uploaded the samples to).\n",
     "# You can find the name from the directory structure inm the left nav\n",
-    "root_dir = \"./Users/<your user alias>/featurestore_sample\"\n",
+    "root_dir = \"./Users/<your_user_alias>/featurestore_sample\"\n",
```
[the remark this answers is not in the document]
Comment (runhli) [reply sdk/python/featurestore_sample/notebooks/sdk_only/2. Enable materialization and backfill feature data.ipynb:162]: discussed offline

**Review by runhli — APPROVED** [review]

**Pull request merged**